types/react-mentions-click/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "@typescript-eslint/no-empty-interface": "off"
```

Comment:
Don't globally disable this rule. It's there for a reason. If you need to violate it, document why in-situ